src/handlers/localstorage/uniswap.ts

```diff
@@ -10,18 +18,31 @@ const ASSETS = 'uniswapassets';
 const LIQUIDITY = 'uniswapliquidity';
 const UNISWAP_POSITIONS = 'uniswapPositions';
 const UNISWAP_FAVORITES = 'uniswapFavorites';
+const UNISWAP_FAVORITES_METADATA = 'uniswapFavoritesMetadata';
```

Comment:
separately (not a blocker and does not need to be addressed in this PR): I just noticed that we aren't properly segmenting the "global" localstorage for favorites/favoritesMeta based on `network` (I think editing the favorites in testnets is broken atm, but if it weren't, you could overwrite mainnet favorites/favoritesMeta from testnets)

Reply:
Good call. I made a ticket for this: https://linear.app/rainbow/issue/RNBW-2445/segment-global-localstorage-for-uniswapfavoritesuniswapfavoritesmeta. @jinchung